lib/services/hpcpackage_remain.pm

```diff
@@ -53,17 +51,35 @@ sub compare_pkg {
     $list1 //= '';
     $list2 //= '';
     # Make a list of installed packages and diff against list from installed packages
-    @diffpkg = split('\n', script_output("diff $list1 $list2 | grep -E '^>' | cut -d' ' -f2 |  grep -E '.*-hpc.*' |   grep -vE 'system|module|suse' |  grep -E '.*_[[:digit:]]+_[[:digit:]]+.*-gnu|.*_[[:digit:]]+_[[:digit:]]+.*-hpc' |  grep -vE '.*-static\$' | tee /tmp/diffpkg.txt", proceed_on_failure => 1, timeout => 180));
+    @diffpkg = split('\n', script_output("diff $list1 $list2 | grep -E '^>' | cut -d' ' -f2 |  grep -E '.*-hpc.*' |   grep -vE 'system|module|suse' |  grep -E '.*_[[:digit:]]+_[[:digit:]]+.*-gnu|.*_[[:digit:]]+_[[:digit:]]+.*-hpc' |  grep -vE '.*-static\$'", proceed_on_failure => 1, timeout => 180));
+}
+
+# The release number needs to be stripped off of the package name
+sub del_num {
```

Comment:
this function stripped off not only the sub release number , but also remove the arch part.

Reply:
Yes, It's OK. As they all have same arch. We don't need compare this part.